bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -162,8 +162,12 @@ public void statusUpdated(final Thing thing, ThingStatusInfo thingStatus) {
                 Bridge bridge = (Bridge) thing;
                 for (Thing bridgeThing : bridge.getThings()) {
                     if (thingStatus.getStatus() == ThingStatus.ONLINE) {
-                        ThingStatusInfo statusInfo = ThingStatusInfoBuilder.create(ThingStatus.ONLINE).build();
-                        setThingStatus(bridgeThing, statusInfo);
+                    	ThingStatusInfo bridgeThingStatus = bridgeThing.getStatusInfo(); 
+                    	if (bridgeThingStatus.getStatus() == ThingStatus.ONLINE || 
```

Comment:
FTR, @marcelrv is referring to my comment https://github.com/openhab/openhab2/pull/357#discussion_r38522344.
But this was not about setting statuses for child devices of a bridge, but about setting the bridge status itself.
In the case here, we are iterating over all child devices and I agree that it does not make sense to send an ONLINE status if they are already ONLINE, since we do not even know at this place whether this is true or not.